src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rental = 40;
+  const longTerm = 7;
+  const shortTerm = 3;
+  const totalPrise = days * rental;
```

Comment:
The variable name `totalPrise` might be a typo. Consider renaming it to `totalPrice` to reflect the correct spelling and enhance code readability.